src/Commands/CommandMakeCommand.php

```diff
@@ -36,7 +36,11 @@ class CommandMakeCommand extends GeneratorCommand
 
     public function getDefaultNamespace() : string
     {
-        return $this->laravel['modules']->config('paths.generator.command.path', 'Console');
+        $module = $this->laravel['modules'];
+
+        return ($namespace = $module->config('paths.generator.command.namespace'))
```

Comment:
Why catching this in a `namespace` variable? I think you can use `?:` instead. In order to have it fit on one line this can be done:

```php
    public function getDefaultNamespace() : string
    {
        $module = $this->laravel['modules'];

        $namespace = $module->config('paths.generator.command.namespace');
        
        return $namespace ?: $module->config('paths.generator.command.path', 'Console');
    }
```
(same for all)

Reply:
Copy will add unit and some changes on it